ios/PurchasesHybridCommon/PurchasesHybridCommon/RCCommonFunctionality.m

```diff
@@ -207,12 +182,12 @@ + (void)purchaseProduct:(NSString *)productIdentifier
         } else {
             completion(@{
                            @"purchaserInfo": purchaserInfo.dictionary,
```

Comment:
I just found this one in Android. Don't forget to update it to `customerInfo`

Reply:
Doh, good catch! Thank you 😊